beacon_node/network/src/sync/range_sync/chain.rs

```diff
@@ -737,6 +740,16 @@ impl<T: BeaconChainTypes> SyncingChain<T> {
 
     /// Returns the next required batch from the chain if it exists. If there are no more batches
     /// required, `None` is returned.
+    ///
+    /// Batches are downloaded excluding the first block of the epoch assuming it has already been
+    /// downloaded.
+    ///
+    /// For example:
+    ///
+    ///
+    /// Epoch boundary |                                   |
+    /// | 30 | 31 | 32 | 33 | 34 | ... | 61 | 62 | 63 | 64 | 65 |
```

Comment:
The epoch boundary should be between 31 and 32 in this diagram (and likewise for higher slots)

Reply:
Oh yeah nice. Bad spacing. Will update